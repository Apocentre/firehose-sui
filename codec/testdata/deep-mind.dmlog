DMLOG BLOCK_BEGIN 8
DMLOG BLOCK_DATA CAgSQDJjNjI0MjMyY2RkMjIxNzcxMjk0ZGZiYjMxMGFjYTAwMGEwZGY2YWM4YjY2YjY5NmQ5MGVmMDZmZGVmYjY0YTMaQDc5MDI2OTliZTQyYzhhOGU0NmZiYmI0NTAxNzI2NTE3ZTg2YjIyYzU2YTE4OWY3NjI1YTZkYTQ5MDgxYjI0NTEgwOf1n+D8h+sWKm4KCHRyYW5zZmVyEkAyZjIyYTgzZTlhMDllNmQxNzNkODg2MDVmODI2N2FiOTZjMzMxODAxNGU0M2RiZmY5NjY0ODg1ZDUxNmRlMDJhGgoweERFQURCRUVGIgoweEJBQUFBQUFEKgAyBAoCJxA4ASp0Cgh0cmFuc2ZlchJAY2VlMmUwMDVjNjRkN2Y0YzI4NjRlODJkOGIxZDcyODczODUyYTIwOWJhNjEzZGU5MTgzODY2OTZmMzQ5ZDE1ZRoKMHhERUFEQkVFRiIKMHhCQUFBQUFBRCoGCgQ7msoAMgQKAicQOAEqdAoIdHJhbnNmZXISQDgyMWJkY2NiNmE3MWE3ZWIyMDAyZjdhYWM3OWY1ODA5OWNkY2FiMzVkZWUwZDU4MmNjNzJhYjMyYTc5MzlkNTUaCjB4REVBREJFRUYiCjB4QkFBQUFBQUQqBgoEdzWUADIECgInEDgBKnQKCHRyYW5zZmVyEkBlODMyZjc3N2QzZjc0ZjZmOGYxZmUwNDViZDgxYTBhNDEzNjc2ZGM0ZjVhMDZhMWE1NmMwNDU5NTlhYThhYTg4GgoweERFQURCRUVGIgoweEJBQUFBQUFEKgYKBLLQXgAyBAoCJxA4ASp0Cgh0cmFuc2ZlchJAYTM3NWViNDliNWE4MWMzMDRhMWIwOWJmYjY1ODE3NDZmZjFkMmFhMWFkYjA3ZWMwYjQ0NTAzMjllMzE5MmIwNhoKMHhERUFEQkVFRiIKMHhCQUFBQUFBRCoGCgTuaygAMgQKAicQOAEqdQoIdHJhbnNmZXISQGYyZDg3NGNmMmEzZjczNDFjZmRkMmM3MDdiNTFkZGNjZDVhMzYyMzkxM2E5NjhmMTFmZTUxY2M4ZDRmY2IxYzAaCjB4REVBREJFRUYiCjB4QkFBQUFBQUQqBwoFASoF8gAyBAoCJxA4ASp1Cgh0cmFuc2ZlchJAZmRiNjE2ZDNlZjI1NTBiNTYzMzJkNjRjOGI4YzczYzEwYmU2Njc5YWZkMzAwNjgwMDc3MzkyOTIwNWEzNTkxZRoKMHhERUFEQkVFRiIKMHhCQUFBQUFBRCoHCgUBZaC8ADIECgInEDgBKnUKCHRyYW5zZmVyEkBlN2NmM2RlMDc5NmZmMWFmMjI4ZjdjZTJhNWIyNjIwY2JkODQzOTI5YTk3YzQwMDIyZTk2NDFhZTQ5N2U4ZGY0GgoweERFQURCRUVGIgoweEJBQUFBQUFEKgcKBQGhO4YAMgQKAicQOAE=
DMLOG BLOCK_END 8