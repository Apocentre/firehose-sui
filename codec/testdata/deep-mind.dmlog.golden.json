[{
  "height": 8,
  "hash": "2c624232cdd221771294dfbb310aca000a0df6ac8b66b696d90ef06fdefb64a3",
  "prevHash": "7902699be42c8a8e46fbbb4501726517e86b22c56a189f7625a6da49081b2451",
  "timestamp": 1645537786620376000,
  "transactions": [
    {
      "type": "transfer",
      "hash": "2f22a83e9a09e6d173d88605f8267ab96c3318014e43dbff9664885d516de02a",
      "sender": "0xDEADBEEF",
      "receiver": "0xBAAAAAAD",
      "amount": {},
      "fee": {
        "bytes": "JxA="
      },
      "success": true
    },
    {
      "type": "transfer",
      "hash": "cee2e005c64d7f4c2864e82d8b1d72873852a209ba613de918386696f349d15e",
      "sender": "0xDEADBEEF",
      "receiver": "0xBAAAAAAD",
      "amount": {
        "bytes": "O5rKAA=="
      },
      "fee": {
        "bytes": "JxA="
      },
      "success": true
    },
    {
      "type": "transfer",
      "hash": "821bdccb6a71a7eb2002f7aac79f58099cdcab35dee0d582cc72ab32a7939d55",
      "sender": "0xDEADBEEF",
      "receiver": "0xBAAAAAAD",
      "amount": {
        "bytes": "dzWUAA=="
      },
      "fee": {
        "bytes": "JxA="
      },
      "success": true
    },
    {
      "type": "transfer",
      "hash": "e832f777d3f74f6f8f1fe045bd81a0a413676dc4f5a06a1a56c045959aa8aa88",
      "sender": "0xDEADBEEF",
      "receiver": "0xBAAAAAAD",
      "amount": {
        "bytes": "stBeAA=="
      },
      "fee": {
        "bytes": "JxA="
      },
      "success": true
    },
    {
      "type": "transfer",
      "hash": "a375eb49b5a81c304a1b09bfb6581746ff1d2aa1adb07ec0b4450329e3192b06",
      "sender": "0xDEADBEEF",
      "receiver": "0xBAAAAAAD",
      "amount": {
        "bytes": "7msoAA=="
      },
      "fee": {
        "bytes": "JxA="
      },
      "success": true
    },
    {
      "type": "transfer",
      "hash": "f2d874cf2a3f7341cfdd2c707b51ddccd5a3623913a968f11fe51cc8d4fcb1c0",
      "sender": "0xDEADBEEF",
      "receiver": "0xBAAAAAAD",
      "amount": {
        "bytes": "ASoF8gA="
      },
      "fee": {
        "bytes": "JxA="
      },
      "success": true
    },
    {
      "type": "transfer",
      "hash": "fdb616d3ef2550b56332d64c8b8c73c10be6679afd3006800773929205a3591e",
      "sender": "0xDEADBEEF",
      "receiver": "0xBAAAAAAD",
      "amount": {
        "bytes": "AWWgvAA="
      },
      "fee": {
        "bytes": "JxA="
      },
      "success": true
    },
    {
      "type": "transfer",
      "hash": "e7cf3de0796ff1af228f7ce2a5b2620cbd843929a97c40022e9641ae497e8df4",
      "sender": "0xDEADBEEF",
      "receiver": "0xBAAAAAAD",
      "amount": {
        "bytes": "AaE7hgA="
      },
      "fee": {
        "bytes": "JxA="
      },
      "success": true
    }
  ]
}
]